src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
-body {
+* {
```

Comment:
Instead of applying styles to all elements using '*', consider targeting only the 'body' element to avoid potential issues with other elements.